app/mapping/beneficiaries/CharityBeneficiaryExtractor.scala

```diff
@@ -93,18 +99,40 @@ class CharityBeneficiaryExtractor @Inject() extends PlaybackExtractor[Option[Lis
         // Assumption that user answered yes as the share of income is not provided
         answers.set(CharityBeneficiaryDiscretionYesNoPage(index), true)
     }
+    } else {
+      Try(answers)
+    }
+  }
+
+  private def extractCountryOfResidence(charityBeneficiary: DisplayTrustCharityType, index: Int, answers: UserAnswers) = {
+    charityBeneficiary.countryOfResidence match {
+      case Some("GB") =>
```

Comment:
Can we store "GB" as a constant somewhere? It's going to be used a lot across all the extractors/mappers